### DISCUSSION TITLE
is LiveReload componet is deprecated soon ?

### DISCUSSION THREAD

**Discussion by stephanvebrian** [remix-run/remix#9455] [Q&A]
is `<LiveReload />` componet is deprecated ?

just spinning up my local development and found this message in the console
```text
@remix-run_react.js?v=cd6e9b06:2856 `<LiveReload />` is obsolete when using Vite and can conflict with Vite's built-in HMR runtime.

Remove `<LiveReload />` from your code and instead only use `<Scripts />`.
Then refresh the page to remove lingering scripts from `<LiveReload />`
```

i cant found in docs stated its depreacated (cmiiw, might be missed it)

**Comment by bluefire2121** [comment]
It's not needed if you're using Vite. See the reference to HMR & HDR

https://remix.run/docs/en/main/guides/vite#hmr--hdr
